addons/web/views/report_templates.xml

```diff
@@ -67,6 +67,16 @@
                     <t t-call-assets="web_editor.summernote" t-css="false"/>
                     <t t-call-assets="web_editor.assets_editor" t-css="false"/>
                     <t t-call-assets="web.report_assets_editor" t-css="false"/>
+                    <script>
```

Comment:
@sle-odoo Ping ! 

Original explanation for this: atp-odoo on Mar 20 Member
got CSRF validation failed on path '/web_editor/attachment/add',

we cannot able to upload local image... without this.